app/views/forms/view/view-contact.directive.html

```diff
@@ -78,15 +78,16 @@ <h4>
 			<address class="km-value">
 				<div class="km-pre">{{document.address | lstring:locale}}</div>
 				{{document.city | lstring:locale}}
-				<span ng-show="document.city">, </span>
+				<span ng-show="document.city">,</span>
 				{{document.state | lstring:locale}}
```

Comment:
{{document.state | lstring:locale}}
```suggestion
				<span ng-bind="document.state | lstring:locale"></span>
```